docs/api_v4.md

```diff
@@ -46,6 +46,11 @@ If a `historic=1` param is included in the request, one additional field will be
 |-------------:|:---------------------------------------------|:------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `history`    | array of numbers                             | never                                                 | Ordered durations of all previous runs. The first item is the first run recorded by the runner's timer into the source file. The last item is the most recent one. This field is only nonempty if the source timer records history.           |
 
+If an `Accept` header is present, splits.io will try to render the run file natively not in JSON of the type specified.  The format for the header is `application/timer`, with timer being any of the timers that can be exported by splits.io.
+If the `Accept` header is valid, the `Content-type` header in the response will be in the format `application/timer`, with timer being the specified timer.  If the `Accept` header is not valid, the `Content-type` header of the response
```

Comment:
I suggest using "`application/<timer>`" over "`application/timer`" to avoid confusion about if the literal `application/timer` is an acceptable header. Then you can refer to `<timer>` later in the sentence to describe what it should be replaced with (like you're doing).